modules/dxp/apps/portal-search-tuning/portal-search-tuning-blueprints-admin-web/src/main/resources/META-INF/resources/js/edit_blueprint/index.js

```diff
@@ -366,10 +359,27 @@ function EditBlueprintForm({
 		})
 			.then((response) => response.json())
 			.then((responseContent) => {
-				setPreviewInfo({
-					loading: false,
-					results: responseContent,
-				});
+				if (formik.isValid) {
+					setPreviewInfo({
+						loading: false,
+						results: responseContent,
+					});
+				}
+				else {
+					setPreviewInfo({
+						loading: false,
+						results: {
+							...responseContent,
+							warnings: [
+								{
+									msg: Liferay.Language.get(
+										'the-configuration-has-missing-or-invalid-values'
+									),
+								},
+							],
+						},
+					});
+				}
```

Comment:
Since only the `warnings` array is being changed with `formik.isValid`, can we move it to where `warnings` is being defined?

```javascript
warnings: formik.isValid ?
  [
     {
        msg: Liferay.Language.get(
		  'the-configuration-has-missing-or-invalid-values'
		),
     }
  ]
  : []
```

Reply:
Will do!